lib/pages/contacts_tab/contacts_tab_body_view.dart

```diff
@@ -16,78 +16,75 @@ class ContactsTabBodyView extends StatelessWidget {
-                builder: (context, value, child) => value.fold(
-                  (failure) => const EmptyContactBody(),
-                  (success) {
-                    if (success is! PresentationContactsSuccess) {
-                      return const LoadingContactWidget();
-                    }
+    return Padding(
+      padding: const EdgeInsets.all(4.0),
+      child: controller.contactsNotifier == null
+          ? null
+          : ValueListenableBuilder(
+              valueListenable: controller.contactsNotifier!,
+              builder: (context, value, child) => value.fold(
+                (failure) => const EmptyContactBody(),
+                (success) {
+                  if (success is! PresentationContactsSuccess) {
+                    return const LoadingContactWidget();
+                  }
 
-                    if (success.data.isEmpty) {
-                      if (success.keyword.isEmpty) {
-                        return const EmptyContactBody();
-                      } else {
-                        return Padding(
-                          padding: const EdgeInsets.only(left: 8.0, top: 8.0),
-                          child: NoContactsFound(
-                            keyword: success.keyword,
-                          ),
-                        );
-                      }
+                  if (success.data.isEmpty) {
+                    if (success.keyword.isEmpty) {
+                      return const EmptyContactBody();
+                    } else {
+                      return Padding(
+                        padding: const EdgeInsets.only(left: 8.0, top: 8.0),
+                        child: NoContactsFound(
+                          keyword: success.keyword,
+                        ),
+                      );
                     }
+                  }
 
-                    return Padding(
-                      padding: const EdgeInsets.symmetric(horizontal: 4.0),
-                      child: Column(
-                        crossAxisAlignment: CrossAxisAlignment.start,
-                        children: success.data
-                            .map<Widget>(
-                              (contact) => InkWell(
-                                borderRadius: BorderRadius.circular(12),
-                                onTap: () {
-                                  controller.onContactTap(
-                                    context: context,
-                                    path: 'rooms',
-                                    contact: contact,
-                                  );
-                                },
-                                child: ExpansionContactListTile(
+                  return Padding(
+                    padding: const EdgeInsets.symmetric(horizontal: 4.0),
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: success.data
+                          .map<Widget>(
```

Comment:
please use SliverList for that, also the load more indicator
![Screenshot 2023-10-02 at 12 28 55](https://github.com/linagora/twake-on-matrix/assets/12546908/65657617-33c8-43d8-a095-8c57c18918e5)
also move the scrollview to the body view

Reply:
Why need to use SilverList and load more indicator already have.

Reply:
it is implemented wrong
all components should in one scroll view only, so the parent is CustomeScrollView, the child is Slivers